test/jdk/javax/swing/JMenuItem/8031573/bug8031573.java

```diff
@@ -59,6 +60,7 @@ public class bug8031573 {
             + "1. Run the test on HiDPI Display.\n"
             + "2. Open the Menu.\n"
             + "3. Check that the icon on the JCheckBoxMenuItem is smooth.\n"
+            + "4. Check that the icon on the JRadioButtonMenuItem is smooth.\n"
             + "   If so, press PASS, else press FAIL.\n";
 
     public static void main(String args[]) throws Exception {
```

Comment:
```suggestion
    public static void main(String[] args) throws Exception {
```
To fix the IDEA warning.